### DISCUSSION TITLE
Keyboard Accessibility Options

### DISCUSSION THREAD

**Discussion by sicsol** [tailwindlabs/headlessui#700] [Ideas]
It would be great to give the ability to `turn off` specific keyboard events from the accessibility/feature list of the components.

As an example not to allow `SPACE` to select an item in `Listbox` for example in our particular case we would like to implement our own `search` functionality however every-time we use the `spacebar` inside the input it closes the menu and selects an item.

**Comment by ADTC** [comment]
See #820
